sky/server/stream_utils.py

```diff
@@ -154,10 +152,7 @@ async def flush_buffer() -> AsyncGenerator[str, None]:
 
         line: Optional[bytes] = await f.readline()
         if not line:
-            if (request_id is not None and
-                    current_time - last_refresh_time >= _REFRESH_INTERVAL):
-                # Refresh the request status from DB every _REFRESH_INTERVAL
-                last_refresh_time = current_time
+            if request_id is not None:
```

Comment:
This is likely causing the largest delay in the benchmark. Not sure how much difference the 0.1 and 0.5 above causes.